aidants_connect_web/templates/aidants_connect_web/new_mandat/recap.html

```diff
@@ -21,7 +21,7 @@ <h1>Récapitulatif</h1>
       {% endif %}
       {% csrf_token %}
       <div class="form__group">
-        <p id="recap_text"><strong>{{ usager }}</strong> autorise <strong>{{ aidant }}</strong>,  <em>aidant·e numérique</em> au sein de <em>nom de l’organisme</em>{{ aidant.organisme }} à réaliser en mon nom conformément aux dispositions des articles 1984 et suivants du Code civil, la ou les démarches administratives en ligne suivantes :</p>
+        <p id="recap_text"><strong>{{ usager }}</strong> autorise <strong>{{ aidant }}</strong>,  <em>aidant·e numérique</em> au sein de <em>nom de l’organisme</em> {{ aidant.organisme }} à réaliser en mon nom conformément aux dispositions des articles 1984 et suivants du Code civil, la ou les démarches administratives en ligne suivantes :</p>
```

Comment:
We should be able to remove `<em>nom de l’organisme</em> ` now that we have it stored!

Reply:
Nice catch 🐠 !